bitfusion_device_plugin/webhook/pkg/webhook/webhook.go

```diff
@@ -374,13 +361,85 @@ func updateAnnotation(target map[string]string, added map[string]string) (patch
 	return patch
 }
 
+func updateInitContainersResources(target, added []corev1.Container) []corev1.Container {
+	maxCpu := zeroQuantity
```

Comment:
the quantity of cpu and memory is set to 0 is valid which means they can have unlimited resource.